### ISSUE TITLE
channelCreate Event fires twice

### ISSUE THREAD

**Issue by LewisHogan** [hydrabolt/discord.js#698]
For example, using the following code will result in the channel creation message being printed to the console twice.
```js
'use strict';
const Discord = require('discord.js');
let bot = new Discord.Client();

bot.on('channelCreate', (channel) => {
  console.log(`New channel has been created ${channel.name} <#${channel.id}>`);
});

bot.login('TOKEN_HERE');
```

**Comment by GusCaplan** [comment]
i was unable to reproduce this, could you provide details on the specific version of discord.js you are using?

**Comment by LewisHogan** [comment]
Running `discord.js@9.2.0` on Node `6.4.0`

**Comment by hydrabolt** [comment]
This has been fixed in f7e89f61e678396e2cb2f8bdd5d1f7bfe029de53 and will be released officially as part of the `9.2.1` release. You can get this release earlier, however, by installing discord.js from the indev branch using `npm i hydrabolt/discord.js#indev`.
